src/Resources/views/PageAdmin/compose.html.twig

```diff
@@ -54,7 +54,7 @@ file that was distributed with this source code.
                                 {% if container.block is defined and container.block %}
                                     <a class="page-composer__page-preview__container block-preview-{{ container.block.id }}"
                                        data-block-id="{{ container.block.id }}"
-                                       data-block-whitelist="{{ container.area.blocks|join(',') }}"
+                                       data-block-allowlist="{{ container.area.blocks|join(',') }}"
```

Comment:
```suggestion
                                       data-block-allowlist="{{ container.area.blocks|join(',') }}"
                                       {# NEXT_MAJOR: Remove the following line #}
                                       data-block-whitelist="{{ container.area.blocks|join(',') }}"
```
It's very unlikely to happen, but just in case someone is using this attribute, we could keep it until next major.

Reply:
I see, that makes sense. Should I change the pull request to master?